alphastats/llm/uniprot_utils.py

```diff
@@ -215,43 +210,69 @@ def extract_data(data: Dict) -> Dict:
     ]
     extracted["tissueSpecificity"] = tissue_specificities
 
-    # 11. Protein Features
-    features = [
-        {
-            "type": feature["type"],
-            "description": feature["description"],
-            "location_start": feature["location"]["start"]["value"],
-            "location_end": feature["location"]["end"]["value"],
-        }
-        for feature in data.get("features", [])
-    ]
-    extracted["features"] = features
-
-    # 12. References
-    references = [
-        {
-            "authors": ref["citation"].get("authors", []),
-            "title": ref["citation"].get("title", ""),
-            "journal": ref["citation"].get("journal", ""),
-            "publicationDate": ref["citation"].get("publicationDate", ""),
-            "comments": [c["value"] for c in ref.get("referenceComments", [])],
-        }
-        for ref in data.get("references", [])
-    ]
-    extracted["references"] = references
+    ## 11. Protein Features
```

Comment:
what about this code? can it be deleted? does it need a reimplementation (then please add a TODO)?

Reply:
I would propose deleting these larger commented blocks. If nobody objects, I will delete them before merging.

Reply:
please delete it right away as removing them it is part of your change request